pkg/transformers/json/transformer.go

```diff
@@ -45,21 +54,32 @@ func transformer(msg messaging.Message) (interface{}, error) {
 		Channel:   msg.Channel,
 		Subtopic:  msg.Subtopic,
 	}
+
 	if ret.Subtopic == "" {
 		return nil, errors.Wrap(ErrTransform, errUnknownFormat)
 	}
+
 	subs := strings.Split(ret.Subtopic, ".")
 	if len(subs) == 0 {
 		return nil, errors.Wrap(ErrTransform, errUnknownFormat)
 	}
+
 	format := subs[len(subs)-1]
 	var payload interface{}
 	if err := json.Unmarshal(msg.Payload, &payload); err != nil {
 		return nil, errors.Wrap(ErrTransform, err)
 	}
+
 	switch p := payload.(type) {
 	case map[string]interface{}:
 		ret.Payload = p
+
+		// Apply timestamnp transformation rules depending on key/unit pairs
+		ts, err := transformTimestamp(msg.Payload)
+		if ts != 0 && err == nil {
+			ret.Created = ts
```

Comment:
you must take care of `jsonPoints` in writers for timestamp to be properly saved in db